bin/.travis/prepare_ezpublish.sh

```diff
@@ -5,6 +5,12 @@
 echo "> Setup github auth key to not reach api limit"
 ./bin/.travis/install_composer_github_key.sh
 
+echo "> Add legacy-bridge to requirements"
+composer require --no-update "ezsystems/legacy-bridge:dev-master"
+
+echo "> Add LegacyBundle to Kernel"
+php ./bin/.travis/enablelegacybundle.php
```

Comment:
What command ?

And I'm not sure how, since we have settings in the default files that require legacy, preventing us from running any sf command at this point :(